lib/forms.js

```diff
@@ -72,6 +72,18 @@ exports.create = function (fields, options) {
             return b;
         },
         handle: function (obj, callbacks) {
+            var returnValue = undefined;
+
+            if (!callbacks) {
```

Comment:
```suggestion
            if (!callbacks && typeof Promise === 'function') {
```

that way, in non-Promise-supporting engines, it will have no effect.

Reply:
Maybe this instead then? As eg. the types will now allow for `callbacks` to be undefined, and better to explicitly tell why it fails then rather than  have it fail down below because it can't access eg. the `success` property of it?

```suggestion
            if (!callbacks) {
                if (typeof Promise !== 'function') {
                    throw new Error('Callbacks are required when Promises are unsupported');
                }
```

Reply:
good call, i like that better.